todo_app/templates/index.html

```diff
@@ -11,8 +11,24 @@ <h1 class="display-4">To-Do App</h1>
     <div class="col-md-12">
       <h2>Items</h2>
       <ul class="list-group mb-4">
-        <!-- List the todo items here -->
+        {% for item in items %}
+                <li>
+                    {{ item.title }}
+                </li>
+            {% endfor %}
       </ul>
     </div>
+
+  <form action="/add" method="post" class="add-item">
+  <div class="add-item">
+      <h2> Add a new item</h2>
+    <label for="title">Item: </label>
+    <input type="text" name="title" required>
+  </div>
+  <div class="add-item">
```

Comment:
You have the class `add-item` on this div and the form tag. Classes that don't apply any styling can be confusing and make the code less clean.